Makefile

```diff
@@ -7,4 +7,5 @@ test.unit:
 
 .PHONY: test.integration
 test.integration:
-	@GOFLAGS="-tags=integration_tests" go test -race -v ./test/integration/...
+	@GOFLAGS="-tags=integration_tests" go test -race -v \
+		-covermode=atomic -coverprofile=coverage.out ./...
```

Comment:
Is the change of the package path intentional? It doesn't seem so.

Reply:
This is intentional. It makes sure that the integration tests also run the unit tests ahead of time so that we are reporting one single complete coverage.out.